CMakeLists.txt

```diff
@@ -664,6 +665,20 @@ elseif(MSVC)
 	add_compile_options("/utf-8")
 ENDIF()
 
+# gcc builds support gprof for profiling
+if(NOT CMAKE_CXX_COMPILER_ID MATCHES "GNU")
+	set(STATUS_GPROF ", NOT SUPPORTED ON THIS PLATFORM")
```

Comment:
This feature is compiler-specific, not platform-specific, so this should probably say "not supported with this compiler" instead.

Reply:
done